### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by szachara**
Added windows support

### CODE REVIEW

File: src/Spatie/Browsershot/Browsershot.php [hunk 1]
```diff
@@ -31,7 +31,11 @@ class Browsershot
     public function __construct($binPath = '', $width = 640, $height = 480, $quality = 60, $timeout = 5000, $backgroundColor = null)
     {
         if ($binPath == '') {
-            $binPath = realpath(dirname(__FILE__).'/../../../bin/phantomjs');
+            if (defined('PHP_WINDOWS_VERSION_BUILD')) {
```
Comment (freekmurze) [inline src/Spatie/Browsershot/Browsershot.php]: Could you add a function to determine the `binPath`. Use early returns to avoid `else`.

Comment (szachara) [inline src/Spatie/Browsershot/Browsershot.php]: Of course

**Comment by szachara** [comment]
Hi, a function to determine the binPath added

**Comment by szachara** [comment]
Hi, a function to determine the binPath added @freekmurze

**Comment by freekmurze** [comment]
thanks!

**Pull request merged**